src/projections.jl

```diff
@@ -1,5 +1,11 @@
 # find expression of projections on cones and their derivatives here:
 #   https://stanford.edu/~boyd/papers/pdf/cone_prog_refine.pdf
+# See also
+#   https://github.com/tjdiamandis/ConeProgramDiff.jl/blob/main/cone_ref.pdf
```

Comment:
We should avoid pointing to main branches of repositories, those links can change, break, etc.
If a pointer to another piece of doc is needed, maybe add it directly here, but I think with those you have in method docstrings we should be good

Reply:
Yea, will remove this